live-site/components/judging-section/judging-schedule-welcome/JudgingScheduleWelcome.styles.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+import PrimaryButtonWide from '../../../../shared-ui/components/primary-button/PrimaryButtonWide';
+import { H2 } from '../../../../shared-ui/style/typography';
+import { colors } from '../../../../shared-ui/style/colors';
+import { max } from '../../../../shared-ui/lib/responsive';
+import { StyledDropdown } from '../../../styled-components/SecondarySelect.styles';
+
+const StyledBackButtonContainer = styled.div`
+  padding-top: 7em;
+  padding-left: 2em;
+`;
+
+const StyledBackButton = styled(PrimaryButtonWide)`
+  padding: 0.5em;
+`;
+
+const StyledJudgingScheduleSection = styled.div`
+  padding-top: 3em;
+  padding-left: 10em;
+  padding-right: 10em;
```

Comment:
delete padding-left and right - it's centered perfectly without those styles and actually when we have them it causes it to not be centered on mobile LOL because you already have the justify-content: center

<img width="479" alt="Screenshot 2023-01-05 at 6 32 44 PM" src="https://user-images.githubusercontent.com/59738880/210899925-bddc60b3-6381-4b48-9d1e-f15b69495468.png">